src/style.css

```
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+}
+
+.wrapper {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-img {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+}
+
+.nav-item {
+  margin-right: 20px;
+}
+
+.nav-list > .nav-item:last-child {
+  margin-right: 50px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #000;
+  line-height: 60px;
+}
+
+.nav-link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  background-color: #00acdc;
+  border-radius: 8px;
+  width: 100%;
+  height: 2px;
```

```suggestion
  height: 4px;
```
It should be 4px by design